default.build

```diff
@@ -235,65 +240,75 @@
 		
 	</target>
 
-  <fileset id="nugetfiles.all" basedir="src">
-    <include name="NHibernate/NHibernate.build" />
-  </fileset>
-
-  <target name="nuspec" depends="init nuget.set-properties" description="Create nuspec files">
-    <nant target="nuspec">
-      <buildfiles refid="nugetfiles.all" />
-    </nant>
-  </target>
-  
-  <target name="nuget" depends="init binaries nuget.set-properties nuspec" 
-          description="Creates files for the release on nuget gallery.">
+	<fileset id="nugetfiles.all" basedir="src">
+		<include name="NHibernate/NHibernate.build" />
```

Comment:
These should be built by standard tools (eg nuget via MsBuild's PackageaProject(dont remember exact name of a property))

Reply:
@hazzik This line is an artifact of the #626 pull request needing to build and test within the current framework.  If you could review and approve that first, it would make this pull request easier.

I don't use these NAnt build files in my [CI branch](https://github.com/ngbrown/nhibernate-core/commits/NH-3807-CI) and do exactly that so that I can build for multiple platforms and multiple CI providers (right now CircleCI and AppVeyor).

There seemed to be resistance to switching away from NAnt, so that's why it's in a separate commit (and branch).  I can replace the NAnt build scripts entirely if there is a feeling that the effort would be accepted into the project.

Reply:
See https://github.com/ngbrown/nhibernate-core/blob/1644ce59001378ba18f863f03935eb77b753e0b6/src/NHibernate/NHibernate.csproj#L15 for the use of `GeneratePackageOnBuild` from MsBuild.

Reply:
I've updated #626 so it is now on top of the new #1404 (Use MsBuild for packing .nupkg files) so there will be no more per-project `.build` files and the `.nupkg` files are built as part of the main MsBuild task.

I'm going to rebase this pull request, but it will take some time.  If the pre-requisite pull requests can be reviewed first, that would certainly help.